src/elements/content-picker/ContentPicker.js

```diff
@@ -102,7 +102,7 @@ type State = {
     isUploadModalOpen: boolean,
     rootName: string,
     searchQuery: string,
-    selected: { [string]: BoxItem },
+    selected: Array<BoxItem>,
```

Comment:
Ideally, we should have a single source of truth for the actual item objects. Can this be an array of item ids that we use to look up the item object from `currentCollection`, instead?